plugins/coral-plugin-love/client/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Coral Plugin Love",
+  "slot": "Comment.Detail"
```

Comment:
I like the idea of sticking to a simple convention like camelCase unless there's an already established convention in the community. I know that the variables themselves are traditionally capitalized, but these strings are grey area. I would probably default to camel case here to keep the concept simple.